File: src/main/java/axirassa/services/runners/SmsNotificationServiceRunner.scala

package axirassa.services.runners

import axirassa.services.phone.SmsNotificationService
import axirassa.util.MessagingTools

object SmsNotificationServiceRunner {
  def main(args : Array[String]) {
    val session = MessagingTools.getEmbeddedSession()
    val service = new SmsNotificationService(session)

    service.execute
  }
}

File: src/main/java/axirassa/services/runners/SmsNotificationServiceRunner.java

package axirassa.services.runners;

import org.hornetq.api.core.client.ClientSession;

import axirassa.services.Service;
import axirassa.services.phone.SmsNotificationService;
import axirassa.util.MessagingTools;

public class SmsNotificationServiceRunner {
	public static void main(String[] args) throws Throwable {
		ClientSession session = MessagingTools.getEmbeddedSession();
		Service service = new SmsNotificationService(session);

		service.execute();
	}
}
